satellite/endpoints/config.go

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package endpoints
```

Comment:
catch-all `packages endpoints` tend to become kitchen sinks. also see https://rakyll.org/style-packages/

Reply:
See above comment